ts/features/idpay/code/screens/IdPayCodeDisplayScreen.tsx

```diff
@@ -74,40 +78,45 @@ const IdPayCodeDisplayScreen = () => {
     : I18n.t("global.buttons.close");
 
   return (
-    <LoadingSpinnerOverlay isLoading={isGeneratingCode} loadingOpacity={1}>
-      <TopScreenComponent contextualHelp={emptyContextualHelp}>
-        <ContentWrapper>
-          <H2>{I18n.t("idpay.code.onboarding.header")}</H2>
-          <VSpacer size={16} />
-          <Body color="grey-700" weight="Regular">
-            {I18n.t("idpay.code.onboarding.body1")}
-          </Body>
-          <Body color="grey-700" weight="Bold">
-            {I18n.t("idpay.code.onboarding.bodyBold")}
-          </Body>
-          <LabelLink>{I18n.t("idpay.code.onboarding.bodyCta")}</LabelLink>
-          <VSpacer size={24} />
-          <CodeDisplayComponent code={idPayCode} />
-          <VSpacer size={24} />
-          <Banner
-            color="neutral"
-            pictogramName="help" // security once new DS ver is released
-            size="big"
-            viewRef={bannerRef}
-            title={I18n.t("idpay.code.onboarding.banner.header")}
-            content={I18n.t("idpay.code.onboarding.banner.body")}
+    <>
+      <LoadingSpinnerOverlay isLoading={isGeneratingCode} loadingOpacity={1}>
+        <TopScreenComponent contextualHelp={emptyContextualHelp}>
+          <ContentWrapper>
+            <H2>{I18n.t("idpay.code.onboarding.header")}</H2>
+            <VSpacer size={16} />
+            <Body color="grey-700" weight="Regular">
+              {I18n.t("idpay.code.onboarding.body1")}
+            </Body>
+            <Body color="grey-700" weight="Bold">
+              {I18n.t("idpay.code.onboarding.bodyBold")}
+            </Body>
+            <LabelLink onPress={presentCieBottomSheet}>
+              {I18n.t("idpay.code.onboarding.bodyCta")}
+            </LabelLink>
+            <VSpacer size={24} />
+            <CodeDisplayComponent code={idPayCode} />
+            <VSpacer size={24} />
+            <Banner
+              color="neutral"
+              pictogramName="help" // security once new DS ver is released
```

Comment:
```suggestion
              pictogramName="security"
```
could you please update this? it's not completely in this PR's scope, but it is an update we need to make since the DS has been updated

Reply:
See https://github.com/pagopa/io-app/pull/5073/commits/9a9c32e0ef0ea90b029cf837e786f0a8db2d678e